sql/core/src/test/scala/org/apache/spark/sql/streaming/FlatMapGroupsWithStateSuite.scala

```diff
@@ -1268,258 +858,6 @@ class FlatMapGroupsWithStateSuite extends StateStoreMetricsTest {
     assert(e.getMessage === "The output mode of function should be append or update")
   }
 
-  import testImplicits._
```

Comment:
Lines in L1271 to L1635 (except a single testcase which isn't related) are moved out to FlatMapGroupsWithStateWithInitialStateSuite. L1790 to L1809 are also moved out as they are only referred by L1271 to L1635.